.travis.yml

```diff
@@ -21,12 +21,13 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
+        - ASTROPY_USE_SYSTEM_PYTEST=1
```

Comment:
This PR fixes compatibility with pytest 3.x, but we don't have to use pytest 3.x for **all** the builds, hence why I'm suggesting maybe only adding it to one or two builds.